src/clients/c++/experimental_api_v2/library/http_client.cc

```diff
@@ -91,14 +91,17 @@ GetQueryString(const Headers& query_params)
 }
 
 void
-Base64Encode(char* raw_ptr, size_t raw_size, char** encoded_ptr)
+Base64Encode(
```

Comment:
Make sure that documentation for this clearly states that the returned encoded_ptr is not null terminated and that encoded_size must be used to determine length of the string.